services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -196,6 +213,11 @@ async def exchange_consumer(
             instrumentation_message_parser,
             {"no_ack": False},
         ),
+        (
+            comp_settings.rabbit.channels["events"],
+            events_message_parser,
+            {"no_ack": False},
```

Comment:
I am not sure if no_ack applies here. but for now I guess that is ok.

Reply:
Is this related to guaranteed delivery of the message?

Reply:
it is related to how you define the exchange. whether rabbitmq will expect an ack or not.
I guess in this specific case we do not care. but that might change. defaults is to have ack